README.md

```diff
@@ -44,8 +48,9 @@ var add = (x, y) => {
   return x + y
 }
 ```
-
--- YOUR ANSWER HERE --
+es6
+no
```

Comment:
This is anonymous. It's just like the one you got write above. All arrow functions are anonymous by definition.